flake.lock: Update

hrp2-14 opened the pull request:
Automated changes by the [update-flake-lock](https://github.com/DeterminateSystems/update-flake-lock) GitHub Action.

```
Flake lock file updates:

• Updated input 'flake-parts':
    'github:hercules-ci/flake-parts/4524271' (2025-09-01)
  → 'github:hercules-ci/flake-parts/5792860' (2026-02-02)
• Updated input 'flake-parts/nixpkgs-lib':
    'github:nix-community/nixpkgs.lib/a73b9c7' (2025-08-10)
  → 'github:nix-community/nixpkgs.lib/7271616' (2026-02-01)
• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/ab0f360' (2025-09-10)
  → 'github:NixOS/nixpkgs/d6c7193' (2026-02-08)
```

### Running GitHub Actions on this PR

GitHub Actions will not run workflows on pull requests which are opened by a GitHub Action.

**To run GitHub Actions workflows on this PR, close and re-open this pull request.**

nim65s committed:
nix: switch to buildStandalone

nim65s commented:
```
pixi install -e test-pixi-build --locked
  /Users/runner/.pixi/bin/pixi install -e test-pixi-build --locked --manifest-path /Users/runner/work/coal/coal/pixi.toml --color always
  Error:   × lock-file not up-to-date with the workspace
```
so I pushed a button on https://github.com/coal-library/coal/actions/workflows/update_pixi_lockfile.yml

github-actions[bot] committed:
flake.lock: Update

Flake lock file updates:

• Updated input 'flake-parts':
    'github:hercules-ci/flake-parts/4524271' (2025-09-01)
  → 'github:hercules-ci/flake-parts/5792860' (2026-02-02)
• Updated input 'flake-parts/nixpkgs-lib':
    'github:nix-community/nixpkgs.lib/a73b9c7' (2025-08-10)
  → 'github:nix-community/nixpkgs.lib/7271616' (2026-02-01)
• Updated input 'nixpkgs':
    'github:NixOS/nixpkgs/ab0f360' (2025-09-10)
  → 'github:NixOS/nixpkgs/d6c7193' (2026-02-08)

nim65s committed:
nix: clean
